src/apis/document_apis.jsx

```diff
@@ -0,0 +1,24 @@
+import AllDocs from "@/apis/test_documents";
+
+/*
+    Gets all information on all docs
+*/
+function GetAllDocs() {
```

Comment:
nit no capital? Also do you think we'll need this api that returns all documents and their bodies anywhere?

Reply:
Not sure, but we can remove it later if need be